adapters/nativo/nativo.go

```diff
@@ -0,0 +1,102 @@
+package nativo
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+// Builder builds a new instance of the Nativo adapter for the given bidder with the given config.
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	bidder := &adapter{
+		endpoint: config.Endpoint,
+	}
+	return bidder, nil
+}
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, _ *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	requestJSON, err := json.Marshal(request)
+
+	if err != nil {
+		return nil, []error{err}
+	}
+
+	headers := http.Header{}
+	headers.Add("Content-Type", "application/json;charset=utf-8")
+
+	requestData := &adapters.RequestData{
+		Method:  "POST",
+		Uri:     a.endpoint,
+		Body:    requestJSON,
+		Headers: headers,
+		ImpIDs:  openrtb_ext.GetImpIDs(request.Imp),
+	}
+
+	return []*adapters.RequestData{requestData}, nil
+}
+
+func (a *adapter) MakeBids(request *openrtb2.BidRequest, externalRequest *adapters.RequestData, response *adapters.ResponseData) (*adapters.BidderResponse, []error) {
+
+	if response.StatusCode != http.StatusOK {
+		switch response.StatusCode {
+			case http.StatusNoContent:
+				return nil, nil
+			case http.StatusBadRequest:
+				return nil, []error{
+					&errortypes.BadInput{
+						Message: fmt.Sprintf("Unexpected status code: %d.", response.StatusCode),
+					},
+				}
+			default:
+				return nil, []error{
+					&errortypes.BadServerResponse{
+						Message: fmt.Sprintf("Unexpected status code: %d.", response.StatusCode),
+					},
+				}
+		}
+	}
```

Comment:
would recommend using adapter utils as done in Appnexus adapterhttps://github.com/prebid/prebid-
server/blob/3295c4119324f6d33d1d9851872cd9b2e594f3f3/adapters/appnexus/appnexus.go#L164-L176

Reply:
done